### ISSUE TITLE
Timestamp component (via TextLayer)

### ISSUE THREAD

**Issue by RaymondDashWu** [CodeLabs-Hubble-gl/kepler.gl#37]

**Comment by RaymondDashWu** [comment]
![image](https://user-images.githubusercontent.com/33266041/90861770-29affc00-e341-11ea-8208-39e526224eda.png)
Error that you get when you try to use view as part of an array ex:
`
return <DeckGL

      views={views: [
        new MapView({id: 'main', controller: true}),
        new MapView({id: 'minimap', x: 10, y: 10, width: '20%', height: '20%', controller: true}})
      ]}
...
`

**Comment by RaymondDashWu** [comment]
layerFilter function + ids so that only text layer is rendered https://deck.gl/docs/developer-guide/views#rendering-layers-in-multiple-views 

coordinate system for text layer - change to cartesian coords https://deck.gl/docs/developer-guide/coordinate-systems
coordinateOrigin

defining viewStates of text + tile within cartesian (?) coords

**Comment by RaymondDashWu** [comment]
Additional resources
https://deck.gl/docs/api-reference/core/orthographic-view

**Comment by RaymondDashWu** [comment]
Best guess is that my setup of Orthographic View isn't correct
![image](https://user-images.githubusercontent.com/33266041/91759025-d2522d00-eb85-11ea-9ce0-226fc0b710f4.png)
Without Orthographic view

![image](https://user-images.githubusercontent.com/33266041/91759201-1c3b1300-eb86-11ea-9c5a-1264c1f59111.png)
With Orthographic view. Bounds lost

**Comment by RaymondDashWu** [comment]
![image](https://user-images.githubusercontent.com/33266041/91759634-d468bb80-eb86-11ea-9c88-bab373edd0d9.png)
This screenshot better illustrates lost bounds

**Comment by RaymondDashWu** [comment]
look at         const animationConfig = this.mapData.visState.animationConfig;
for actual timestamp data once ready

**Comment by RaymondDashWu** [comment]
Can't use onViewStateChange to make the timestamp stick to a certain view in frame like normal Deck view because Hubble movement isn't done using mouse, touch, keyboard.
https://deck.gl/docs/api-reference/core/deck#onviewstatechange

**Comment by RaymondDashWu** [comment]
![image](https://user-images.githubusercontent.com/33266041/92192176-44db3b00-ee1a-11ea-923c-c9043a5d0d55.png)

¯\_(ツ)_/¯ Add clarification. later

**Comment by RaymondDashWu** [comment]
Possibly consider using mapbox to add timestamp component as those go on top of Deck https://deck.gl/docs/api-reference/mapbox/overview#inserting-a-2d-deck-layer-before-an-existing-mapbox-layer

**Comment by RaymondDashWu** [comment]
TLDR; didn't seem to be possible to do in previous versions of Kepler. Try again in future?
